Fits/Dreadnoughts/[FL33T] Anti-Cap Phoenix Navy Issue.md

```diff
@@ -0,0 +1,53 @@
+# Phoenix Navy Issue
+
+Anti-Capital PNI
+
+## Description
+
+Used to provide high DPS against other capital ships.  Highly ineffective for use against sub-capital targets.  Fuel bay should have 60k Nitrogen Isotopes and be topped up with Strontium Clathrates. Fleet hangar should have an additional 1000 Strontium Clathrates.  The remainder of cargo space in both the fleet hangar and main hold should be filled with Cap Booster 3200s.
+
+## Fit
+```
+[Phoenix Navy Issue, Anti-Cap Phoenix Navy Issue]
```

Comment:
Same comment as the other Phoenix. Also, do we get much out of the T2 capital booster?

Reply:
Shield dread cap is just real touchy, the extra booster before reload is pretty big and it's by far the biggest weakness of shield dreads so the more cap we can get on them the better.